docs/start.md

```diff
@@ -9,6 +9,10 @@ version from PyPI:
 pip install zappend
 ```
 
+You can also install `zappend[dev]` to also install development dependencies,
```

Comment:
I tried to come up with a good suggestion to get rid of the numerous "also install"s in this sentence, but couldn't think of any. Maybe you can rephrase anyway?

Reply:
I managed to get rid of one of them, but I'm keeping the other two (despite ugliness) for maximum clarity.